_vendor/normalize-scss/sass/normalize/_normalize-mixin.scss

```diff
@@ -424,7 +424,7 @@
     optgroup,
     select,
     textarea {
-      font-family: inherit; /* 1 */
+      font-family: $base-font-family; /* 1 */
```

Comment:
I guess this doesn't belong here. Looks like another topic and should be discussed in a separate thread.